app/helpers/route_overrides_helper.rb

```diff
@@ -0,0 +1,40 @@
+module RouteOverridesHelper
+  class << self
+
```

Comment:
This style check is contradicting. No empty line at the beginning of the class vs an empty line before `private`. Which one should take the precedence? >.<